near-sdk/src/lib.rs

```diff
@@ -5,6 +5,8 @@
 #[cfg(test)]
 extern crate quickcheck;
 
+#[cfg(feature = "abi")]
```

Comment:
can we put this export under unstable as well for now? Doesn't seem clear this will be more intuitive than deriving the traits directly yet and the internal semantics aren't tested yet.